### PR TITLE
More 941130 tests

### PR DESCRIPTION
**Pull request by dune73**

### CODE REVIEW

**Comment by dune73** [comment]
More tests for 941130, where we just merged a PR to make it RE2 compatible.

@allanrbo / @ fgs: Care to do a quick review?

File: util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941130.yaml [hunk 1]
```diff
@@ -1,23 +1,278 @@
...
+    desc: "XSS test for 941130"
+    stages:
+    - 
+      stage:
+        input:
+          dest_addr: 127.0.0.1
+          headers:
+            Host: localhost
+            method: POST
+            port: 80
```
Comment (fgsch) [inline util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941130.yaml]: port should be part of input, not headers.

File: util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941130.yaml [hunk 2]
```diff
@@ -1,23 +1,278 @@
...
```
Comment (fgsch) [inline util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941130.yaml]: same here and a couple of others.

File: util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941130.yaml (hunk 1 above)
Comment (fgsch) [reply util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941130.yaml]: method too.

Comment (fgsch) [reply util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941130.yaml]: Can we also ensure that we have User-Agent and maybe Accept too so ideally we're only tripping a single rule?

**Comment by dune73** [comment]
Thank you for the review. This was a quick shot, maybe too quick. :)

Will update.

**Comment by dune73** [comment]
Tests failing due to problem with new test 941110-10 outside this PR.

**Comment by dune73** [comment]
OK. Tests work again, PR is updated. Ready for review.

**Comment by dune73** [comment]
Have not received any feedback after my update. Assuming the feedback above was all to be said. I addressed those points and plan to merge soon.

**Comment by fgsch** [comment]
I will try to take a look over the weekend.

**Comment by lifeforms** [comment]
@fgsch Did you manage to take a look at it? Tests seem to be succeeding, I'd love to merge it.

**Comment by fgsch** [comment]
I had a quick look and looks correct, plus tests are passing 👍

**Pull request merged**